wiki/Installation/id.md

```diff
@@ -1,70 +1,77 @@
 
 ## Persyaratan sistem minimum
 
-- .NET framework 4+ (alat penginstal seharusnya meminta anda untuk mengunduhnya, jika tidak, anda dapat menemukannya [di sini](https://www.microsoft.com/id-id/download/details.aspx?id=48130))
+- .NET framework 4+ (penginstal osu! seharusnya meminta kamu untuk mengunduhnya, jika tidak, kamu dapat menemukannya [di sini](https://www.microsoft.com/id-id/download/details.aspx?id=48130))
 - Kartu grafis yang memadai.
 
 ## Memasang osu!
 
-1. Pergi ke [halaman unduh](https://osu.ppy.sh/home/download) dan unduh penginstal osu!.
+1. Pergi ke [laman unduh](https://osu.ppy.sh/home/download) dan unduh penginstal osu!.
 2. Buka file yang baru saja diunduh dan jalankan penginstal.
-3. Saat penginstal berjalan, akan ada waktu sepuluh detik untuk memungkinkan anda memilih lokasi pemasangan.
-    - Jika anda ingin mengubahnya, klik pada teks di bawah kata, "Hi!" dan pilih lokasi pemasangan.
-    - osu! akan dipasang ke lokasi itu ketika pengatur waktu selesai.
-4. Saat penginstalan selesai, osu! akan mulai tepat setelahnya. osu!direct akan digunakan untuk mengunduh beberapa beatmap starter untuk anda.
-5. osu! akan meminta anda untuk masuk atau mendaftar.
-    - Jika anda sudah memiliki akun, masuklah.
-    - Jika anda perlu mendaftar, lihat [Registrasi](/wiki/Registration), lalu masuk.
+3. Saat penginstal berjalan, akan ada waktu sepuluh detik untuk memungkinkan kamu memilih lokasi pemasangan.
+   - Jika kamu ingin mengubahnya, klik pada teks di bawah kata, `Hi!` dan pilih lokasi pemasangan.
+   - osu! akan dipasang berdasarkan lokasi yang kamu pilih saat waktu mundur telah selesai.
+4. osu! akan terbuka sesaat setelah proses pemasangan selesai. osu!direct akan digunakan untuk mengunduh beberapa beatmap pengantar untukmu.
+5. Terakhir osu! akan meminta kamu untuk masuk atau mendaftar.
+   - Jika kamu sudah memiliki akun, silakan masuk.
+   - Jika kamu perlu mendaftar, lihat artikel [Registrasi](/wiki/Registration), lalu masuk.
+
+## Penyelesaian masalah
+
+Jika kamu menggunakan Windows 7 64-bit, kamu mungkin akan mengalami beberapa kendala saat memasang osu!. Kamu bisa coba memasang .NET Framework versi 64-bit, jika program ini tidak disertakan dalam penginstal osu. [Unduh di sini](https://download.microsoft.com/download/2/0/e/20e90413-712f-438c-988e-fdaa79a8ac3d/dotnetfx35.exe)
+
+---
+
+**Jika penginstal osu! memberikan kode kesalahan saat dipasang, kamu dapat mencoba beberapa hal berikut:**
+
+- Coba pasang ulang .NET Framework dan pastikan untuk menggunakan versi terbaru. [Kamu dapat mengunduhnya lewat tautan berikut](https://dotnet.microsoft.com/download)
+- Bersihkan PC kamu - Pindai dengan anti-virus, bersihkan juga registry-nya. Program yang bagus untuk melakukan pembersihan umum adalah CCleaner. Pindai file komputer dan registry kamu. Pemasang osu! bisa saja gagal dijalankan dikarenakan ada entri registry osu! dari salinan osu! yang masih ada dalam sistem kamu.
```

Comment:
`Penginstal osu! bisa saja gagal dijalankan dikarenakan ada entri registry osu! dari salinan osu! yang masih ada dalam sistem kamu.`

More consistent wording, just like the top part